src/r4300/new_dynarec/assem_arm.c

```diff
@@ -2053,14 +2053,11 @@ static void emit_writebyte(int rt, int addr)
   output_w32(0xe5c00000|rd_rn_rm(rt,FP,0)|offset);
 }
 
-/*
-static void emit_mul(int rs)
+static void emit_mul(u_int rs1,u_int rs2,u_int rt)
 {
-  assem_debug("mul %%%s",regname[rs]);
-  assert(0);
+  assem_debug("mul %s,%s,%s",regname[rt],regname[rs1],regname[rs2]);
+  output_w32(0xe0000090|(rt<<16)|(rs2<<8)|rs1);
```

Comment:
So it's not supposed to change.

Do you mean no one have still think to use something like `#define ARMI_MUL 0xe0000090` or `X86I_MUL` for instruction opcode? Any cons about using define for such thing?

That's easy to do and would make the code even cleaner. Is this relevant to create a ticket for this? :)

C experts, any advise?